_opensearch/snapshots/snapshot-management.md

```diff
@@ -0,0 +1,100 @@
+
+Snapshot management depends on the job scheduler plugin to schedule a job that is run periodically. Each SM policy corresponds to one SM scheduled job. The scheduled job is lightweight, so the burden of SM depends on the snapshot creation frequency and the burden of running snapshot operation itself. However, too many pending SM tasks can block other high-priority tasks in the cluster.
+
+## Concurrency 
+
+- Taking a snapshot doesn’t block indexing or other requests. However, the snapshot won’t include changes made after the snapshot process starts. 
+
+- Snapshot deletion doesn't block snapshot creation.
+
+- Taking a snapshot can temporarily pause shard allocations. 
+
+We don't recommend creating several SM policies with the same schedule and overlapping indices in one cluster, since it hinders performance. 
+{: .warning }
+
+We don't recommend setting up the same repository for multiple SM policies in different clusters, since it hinders performance. 
+{: .warning }
+
+## Failure management
+
+If snapshot creation fails, it is retried for a maximum of three times. The failure message is saved into metadata and is removed if a subsequent snapshot creation attempt succeeds. Possible reasons for failure include red indices, and shard reallocation.
+
+## Security
+
+SM follows the OpenSearch plugin security model:
+
+- The system index is only accessible to super admin (who uses the cluster SSL credential directly). Even the admin user cannot access the system index.
+    
+- The security plugin checks each added API plugin at transport layer. The security plugin checks the security roles of the calling user if the roles contain the permission associated with this API.
+    
+- The API calls within the plugin impersonate the user. For example, SM saves the user security roles, and impersonates the user with these roles when calling the snapshot APIs. 
+
+Security plugin provides role-based access control for OpenSearch. The role associated with user contains [permissions]({{site.url}}{{site.baseurl}}/security-plugin/access-control/permissions) for this user. Security plugin checks the security role of the impersonated user when the snapshot API is called.
+
+To use SM, the user’s role is must have the following permissions:
+
+- SM APIs. For example, the role must have the permission `cluster:admin/opensearch/snapshot_management/put` for SM index policy API. Users without necessary permissions will be blocked when calling the SM API.
+
+- Get/create/delete snapshot APIs. For example, the user must have the permission `cluster:admin/snapshot/get` for get snapshot API. If the user does not have the necessary permissions, SM will fail to perform these snapshot operations at runtime.
+
+The following table lists the required permissions for each function of SM.
```

Comment:
The following table lists the associated permissions for each API of SM.

Reply:
Done